contracts/evm/contracts/adapters/ClusterConnection.sol

```diff
@@ -141,42 +146,65 @@ contract ClusterConnection is Initializable, IConnection {
         string memory srcNetwork,
         uint256 _connSn,
         bytes calldata _msg,
+        string memory dstNetwork,
```

Comment:
This is not needed as a parameter but should preferably be taken from the contract itself

Reply:
Updated